app/views/short_url_requests/show.html.erb

```diff
@@ -6,4 +6,5 @@
   <%= button_to "Accept and create redirect", accept_short_url_request_path(@short_url_request), method: :post, class: "btn btn-success" %>
   <%= link_to "Reject", new_rejection_short_url_request_path(@short_url_request), class: "btn btn-danger" %>
 <% end %>
+<%= link_to "Edit", edit_short_url_request_path, class: "btn btn-default" %>
 <%= link_to "Back", short_url_requests_path, class: "btn btn-default" %>
```

Comment:
Can we remove the back button? There's too many buttons here. Maybe add a crumb trail?